README.md

```diff
@@ -25,18 +25,19 @@
   </p>
 
 <p align="center">
-<a href="https://oshpark.com/shared_projects/erADDDjN"><img src="https://oshpark.com/packs/media/images/badge-5f4e3bf4bf68f72ff88bd92e0089e9cf.png" alt="Order from OSH Park"></img></a>
+<a href="https://www.pcbpower.com/"><img src="./documentation/assets/pcb_power.jpeg"></img></a>
```

Comment:
no image named `pcb_power.jpeg` in the `assets` folder.
Instead can add link to the image from web.